internal/cli/performanceadvisor/namespaces/list.go

```diff
@@ -66,27 +66,27 @@ func (opts *ListOpts) newNamespaceOptions() *atlas.NamespaceOptions {
 	}
 }
 
-func (opts *ListOpts) validateProcessName() error {
+func validateProcessName(processName string) error {
 	const length = 2
-	process := strings.Split(opts.processName, ":")
+	process := strings.Split(processName, ":")
 	if len(process) != length {
-		return fmt.Errorf("'%v' is not valid", opts.processName)
+		return fmt.Errorf("'%v' is not valid", processName)
 	}
 	return nil
 }
 
-func (opts *ListOpts) host() (string, error) {
-	if opts.processName == "" {
-		return opts.hostID, nil
+func Host(processName, hostID string) (string, error) {
```

Comment:
I will leave this as it was before then so that I don't have to add another param to use `opts.store.PerformanceAdvisorSlowQueries(opts.ConfigProjectID(), host, opts.newSlowQueryOptions())`